tools/check_app_environment.py

```diff
@@ -29,7 +32,7 @@ def output(cmd):
         lambda v: v.split("\n")[-1].split()[2],
         "12.0",
     ),
-    "pnpm": (["pnpm", "-v"], lambda v: v, "10.17.1"),
+    js_installer: ([js_installer, "-v"], lambda v: v, "0.0.0"),
```

Comment:
Does this mean the version isn't checked? I'm not sure how to interpret that change